bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -629,6 +613,37 @@ for inj_file, tag in zip(inj_files, inj_tags):
                                 tags=[tag])
 
 
+######################## Make combined injection plots ##########################
+if len(files_for_combined_injfind) > 0:
+    found_inj_comb = wf.find_injections_in_hdf_coinc\
+        (workflow, inj_coincs, files_for_combined_injfind, censored_veto,
+         censored_veto_name, 'allinj', tags=['ALLINJ'])
```

Comment:
should this be censored_veto_name or censored_veto_name[0] ? seems like we want the latter.